tasks/pocket_sand.py

```diff
@@ -0,0 +1,20 @@
+import random
+from discord_bot import STATIC_PATH
+import asyncio
+import os
+import discord
+from tasks.base import BaseTask
+
+
+class DaleAttack(BaseTask):
+    id = 'dale-attack'
+    channel = None
+
+    async def task(self):
+        if not self.channel:
+            self.channel = self.client.get_channel(596772352835190823)
+        while True:
+            attack_time = random.randint(300, 600)
```

Comment:
Should there be an associated command to start and stop this task? Otherwise it would run and keep posting forever even when we are not playing.